src/writeJson.py

```diff
@@ -633,36 +633,70 @@
-                                        'Lawrence Livermore National Laboratory, 25 pp. ',
-                                        'Available online: https://pcmdi.llnl.gov/report/pdf/60.pdf'])
-source_id[key]['region'] = ['global_ocean']
-source_id[key]['release_year'] = '2019'
-source_id[key]['source_description'] = 'Sea surface temperature and sea-ice datasets produced by PCMDI (LLNL) for the AMIP (DECK) experiment of CMIP6'
-source_id[key]['source'] = 'PCMDI-AMIP 1.1.6: Merged SST based on UK MetOffice HadISST and NCEP OI2'
-source_id[key]['source_id'] = key
-source_id[key]['source_type'] = 'satellite_blended'
-source_id[key]['source_variables'] = ['areacello','sftof','siconc','siconcbcs',
-                                      'tos','tosbcs']
-source_id[key]['source_version'] = '1.1.6'
-source_id[key]['target_mip'] = 'CMIP'
-source_id[key]['title'] = 'PCMDI-AMIP 1.1.6 dataset prepared for input4MIPs'
+# Loop over ISMIP6 source_id entries
+sIds = [
+ 'ACCESS1-3-rcp85-1-0',
+ 'CCSM4-rcp26-1-0',
+ 'CCSM4-rcp85-1-0',
+ 'CESM2-ssp585-1-0',
+ 'CNRM-CM6-1-ssp126-1-0',
+ 'CNRM-CM6-1-ssp585-1-0',
+ 'CNRM-ESM2-1-ssp585-1-0',
+ 'CSIRO-MK3-6-0-rcp85-1-0',
+ 'HadGEM2-ES-rcp85-1-0',
+ 'IPSL-CM5A-MR-rcp26-1-0',
+ 'IPSL-CM5A-MR-rcp85-1-0',
+ 'MIROC-ESM-CHEM-rcp26-1-0',
+ 'MIROC-ESM-CHEM-rcp85-1-0',
+ 'MIROC5-rcp26-1-0',
+ 'MIROC5-rcp85-1-0',
+ 'NorESM1-M-rcp26-1-0',
+ 'NorESM1-M-rcp85-1-0',
+ 'UKESM1-0-LL-ssp585-1-0'
+ ]
+
+for count,sId in enumerate(sIds):
+    key = sId
+    tmp = key.replace('-1-0','')
+    rcp = tmp.split('-')[-1]
+    mod = tmp.replace(''.join(['-',rcp]),'')
+    print(count,key,mod,rcp)
+    #sys.exit()
+    source_id[key] = {}
+    source_id[key]['comment'] = ' '.join(['Prepared using CMIP5 model', mod ,
+                                          'as input. A combination of historical',
+                                          'and', rcp ,'datasets were used to',
+                                          'create this ISMIP6 forcing dataset'])
+    source_id[key]['contact'] = 'ISMIP6 Steering Team (ismip6@gmail.com)'
+    source_id[key]['dataset_category'] = 'atmosphericState'
+    source_id[key]['grid'] = '1x1 degree longitude x latitude'
+    source_id[key]['grid_label'] = 'gn'
+    source_id[key]['further_info_url'] = 'http://www.climate-cryosphere.org/wiki/index.php?title=ISMIP6_wiki_page'
+    source_id[key]['institution_id'] = 'NASA-GSFC'
+    source_id[key]['institution'] = 'NASA Goddard Space Flight Center, Greenbelt, MD 20771, USA'
+    source_id[key]['nominal_resolution'] = '1x1 degree'
+    source_id[key]['product'] = 'derived'
+    source_id[key]['references'] = ' '.join(['Experimental protocol for sealevel',
+                                            'projections from ISMIP6 standalone',
+                                            'ice sheet models, Nowicki, S. et al,',
+                                            '2020, https://doi.org/10.5194/tc-2019-322'])
+    source_id[key]['region'] = ['global_ocean']
```

Comment:
"region" will be either "antarctica" or "greenland"

Reply:
Ok, I will remove this from the registration, like the other entries you can add this at run time